packages/zowe-explorer-api/CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ All notable changes to the "zowe-explorer-api" extension will be documented in t
 
 ### New features and enhancements
 
+- Updated the `@zowe/cli` dependency to 7.27.0 to support proxy environment variables [#3003](https://github.com/zowe/zowe-explorer-vscode/issues/3003)
```

Comment:
should this say **v** 7.27.0?

Reply:
I modeled it after previous changelog entries but can change it if the "v" prefix is preferred 😋